### PR TITLE
V1.3.1

### PR DESCRIPTION
**Pull request by chrispyatt**
Updated to use eggd_canrisk_vcf v1.1.1

- - -
This change is [Reviewable](https://reviewable.io/reviews/eastgenomics/eggd_CANRISK_workflow/7)

## Summary by CodeRabbit

- **New Features**
	- Updated version of the CANRISK workflow to 1.3.1.
	- Enhanced `eggd_canrisk_vcf` application version to 1.1.1, including a bug fix.
  
- **Documentation**
	- Revised `README.md` to reflect the new version numbers.
	- Updated `dxworkflow.json` with new version details and a bug fix entry.

### CODE REVIEW

**Review by growland2 — APPROVED** [review]
Looks good, will hold off merging, awaiting sign-off for the canrisk v1.1.1 app and an example successful workflow job

> Reviewed 2 of 2 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/eastgenomics/eggd_CANRISK_workflow/7#-:-ODMc1lq3YAfwj03NGCO:bon5bnt)* status: all files reviewed, 2 unresolved discussions (waiting on @chrispyatt)

**Comment by chrispyatt** [comment]
Successful job: https://platform.dnanexus.com/panx/projects/Gx9GqQ044gkVJKzvzPX1QPq6/monitor/analysis/Gx9GzvQ44gkZpFxkQgXPj46Y

**Review by mattgarner — APPROVED** [review]
> Reviewed 1 of 2 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/eastgenomics/eggd_CANRISK_workflow/7)* status: all files reviewed, 2 unresolved discussions (waiting on @chrispyatt)

**Review by mattgarner — COMMENTED** [review]
> Dismissed @coderabbitai[bot] from 2 discussions.
> *[Reviewable](https://reviewable.io/reviews/eastgenomics/eggd_CANRISK_workflow/7)* status: <span>:shipit:</span> complete! all files reviewed, all discussions resolved (waiting on @chrispyatt)

**Pull request merged**
